
Fix drm trace in logs:
No drm_driver.set_busid() implementation provided by nv_drm_driver [nvidia]

Signed-off-by: Thomas Backlund <tmb@mageia.org>

diff -Nurp NVIDIA-Linux-x86_64-346.35.orig/kernel/nv-drm.c NVIDIA-Linux-x86_64-346.35/kernel/nv-drm.c
--- NVIDIA-Linux-x86_64-346.35.orig/kernel/nv-drm.c
+++ NVIDIA-Linux-x86_64-346.35/kernel/nv-drm.c
@@ -127,6 +127,9 @@ static struct drm_driver nv_drm_driver =
     .gem_prime_get_sg_table = nv_gem_prime_get_sg_table,
     .gem_prime_vmap = nv_gem_prime_vmap,
     .gem_prime_vunmap = nv_gem_prime_vunmap,
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0)
+    .set_busid = drm_pci_set_busid,
+#endif
 
     .name = "nvidia-drm",
     .desc = "NVIDIA DRM driver",
